packages/tangle/scheduler.go

```diff
@@ -104,7 +105,16 @@ func (s *Scheduler) Shutdown() {
 }
 
 // Setup sets up the behavior of the component by making it attach to the relevant events of the other components.
-func (s *Scheduler) Setup() {}
+func (s *Scheduler) Setup() {
+	// pass booked messages to the scheduler
+	s.tangle.Booker.Events.MessageBooked.Attach(events.NewClosure(func(messageID MessageID) {
+		if err := s.SubmitAndReady(messageID); err != nil {
+			s.Events.Error.Trigger(errors.Errorf("failed to submit to scheduler: %w", err))
```

Comment:
An error might be a bit strong here. There are 3 possibilities:

1. `ErrInboxExceeded` this is kind of expected behavior and we are triggering `NodeBlacklisted` anyways. So maybe this isn't even worth to log.
2. `ErrInsufficientMana` will never happen as the min access mana is sufficiently large.
3. Any other error coming from the `ObjectStorage` etc. These are probably actual errors.